### PR TITLE
support edge in prisma adapter

### PR DESCRIPTION
**Pull request by shiyuhang0**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 1]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
 
 [Prisma](https://www.prisma.io/docs) is an open source next-generation ORM (Object-Relational Mapping) that helps developers interact with their database in an intuitive, efficient, and safe way. TiDB Cloud offers [@tidbcloud/prisma-adapter](https://github.com/tidbcloud/prisma-adapter), enabling you to use [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) over HTTPS with [TiDB Cloud serverless driver](/tidb-cloud/serverless-driver.md). Compared with the traditional TCP way, [@tidbcloud/prisma-adapter](https://github.com/tidbcloud/prisma-adapter) brings the following benefits:
 
-- Better performance in serverless environments
-- Possibility of using Prisma client in the edge environments (see [#21394](https://github.com/prisma/prisma/issues/21394) for more information)
+- Better performance in the serverless environments
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:10]:
```suggestion
- Better performance of Prisma Client in serverless environments
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 2]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+- Ability to use in the edge environments
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:11]:
```suggestion
- Ability to use Prisma Client in edge environments
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 3]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
 
-This tutorial describes how to use TiDB Cloud serverless driver with the Prisma adapter.
+This tutorial describes how to use the Prisma adapter.
+
+## Install
+
+You need to install both the `prisma-adapter` and `serverless driver`. You can install them using npm or your preferred package manager.
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:17]:
```suggestion
You need to install both [@tidbcloud/prisma-adapter](https://github.com/tidbcloud/prisma-adapter) and [TiDB Cloud serverless driver](/tidb-cloud/serverless-driver.md). You can install them using [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) or your preferred package manager.

Taking npm as an example, you can run the following commands for installation: 

```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 4]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+
+datasource db {
+  provider     = "mysql"
+  url          = env("DATABASE_URL")
+}
+```
+
+## Initialize the Prisma Client
+
+After initialize the Prisma Client with the Prisma adapter, you can use the Prisma Client as usual. Then, the query will go through the TiDB Cloud serverless driver. Here is an example:
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:42]:
```suggestion
## Initialize Prisma Client

Before using Prisma Client, you need to initialize it with `@tidbcloud/prisma-adapter`. For example:
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 5]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+
+// Initialize Prisma Client
+const connection = connect({ url: ${DATABASE_URL} });
+const adapter = new PrismaTiDBCloud(connection);
+const prisma = new PrismaClient({ adapter });
+```
 
 ## Use the Prisma adapter in Node.js environments
 
+This is an example of how to use the Prisma adapter in Node.js environments.
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:57]:
```suggestion
This section provides an example of how to use `@tidbcloud/prisma-adapter` in Node.js environments.
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 6]
```diff
@@ -216,4 +258,7 @@ To complete this tutorial, you need the following:
    
 ## Use the Prisma adapter in edge environments
 
-Currently, `@tidbcloud/prisma-adapter` is not compatible with edge environments such as Vercel Edge Function and Cloudflare Workers. However, there are plans to support these environments. For more information, see [#21394](https://github.com/prisma/prisma/issues/21394).
\ No newline at end of file
+You can use `@tidbcloud/prisma-adapter` >= v5.11.0 in the edge environments such as Vercel Edge Function and Cloudflare Workers.
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:261]:
```suggestion
You can use `@tidbcloud/prisma-adapter` v5.11.0 or a later version in edge environments such as Vercel Edge Function and Cloudflare Workers.
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 7]
```diff
@@ -216,4 +258,7 @@ To complete this tutorial, you need the following:
...
+
+- [Vercel Edge Function Example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-vercel-example)
+- [Cloudflare Worker Example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-cloudflare-worker-example)
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:264]:
```suggestion
- [Vercel Edge Function example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-vercel-example)
- [Cloudflare Worker example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-cloudflare-worker-example)
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 8]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
 
 [Prisma](https://www.prisma.io/docs) is an open source next-generation ORM (Object-Relational Mapping) that helps developers interact with their database in an intuitive, efficient, and safe way. TiDB Cloud offers [@tidbcloud/prisma-adapter](https://github.com/tidbcloud/prisma-adapter), enabling you to use [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) over HTTPS with [TiDB Cloud serverless driver](/tidb-cloud/serverless-driver.md). Compared with the traditional TCP way, [@tidbcloud/prisma-adapter](https://github.com/tidbcloud/prisma-adapter) brings the following benefits:
 
-- Better performance in serverless environments
-- Possibility of using Prisma client in the edge environments (see [#21394](https://github.com/prisma/prisma/issues/21394) for more information)
+- Better performance in the serverless environments
+- Ability to use in the edge environments
 
-This tutorial describes how to use TiDB Cloud serverless driver with the Prisma adapter.
+This tutorial describes how to use the Prisma adapter.
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:13]:
```suggestion
This tutorial describes how to use [@tidbcloud/prisma-adapter](https://github.com/tidbcloud/prisma-adapter) in serverless environments and edge environments.
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 9]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+## Install
+
+You need to install both the `prisma-adapter` and `serverless driver`. You can install them using npm or your preferred package manager.
+
+```
+npm install @tidbcloud/prisma-adapter
+npm install @tidbcloud/serverless
+```
+
+## Enable the driverAdapters
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:24]:
```suggestion
## Enable `driverAdapters`
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 10]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+
+To use the Prisma adapter, we need to enable the `driverAdapters` feature in the `schema.prisma` file. Here is an example:
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:26]:
```suggestion
To use the Prisma adapter, you need to enable the `driverAdapters` feature in the `schema.prisma` file. For example:
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 11]
```diff
@@ -7,13 +7,55 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+```
+import { connect } from '@tidbcloud/serverless';
+import { PrismaTiDBCloud } from '@tidbcloud/prisma-adapter';
+import { PrismaClient } from '@prisma/client';
+
+// Initialize Prisma Client
+const connection = connect({ url: ${DATABASE_URL} });
+const adapter = new PrismaTiDBCloud(connection);
+const prisma = new PrismaClient({ adapter });
+```
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:54]:
```suggestion

Then, queries from Prisma Client can be sent to the TiDB Cloud serverless driver for processing. 

```

File: tidb-cloud/serverless-driver-prisma-example.md (hunk 6 above)
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:261]:
```suggestion
You can use `@tidbcloud/prisma-adapter` >= v5.11.0 in edge environments such as Vercel Edge Function and Cloudflare Workers.
```

File: tidb-cloud/serverless-driver-prisma-example.md (hunk 7 above)
Comment (qiancai) [inline tidb-cloud/serverless-driver-prisma-example.md:264]:
```suggestion
- [Vercel Edge Function example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-vercel-example)
- [Cloudflare Worker example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-cloudflare-worker-example)
```

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 12]
```diff
@@ -7,13 +7,59 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+npm install @tidbcloud/prisma-adapter
+npm install @tidbcloud/serverless
+```
+
+## Enable `driverAdapters`
+
+To use the Prisma adapter, you need to enable the `driverAdapters` feature in the `schema.prisma` file. For example:
+
+```
```
Comment (Oreoxmt) [inline tidb-cloud/serverless-driver-prisma-example.md:30]:
```suggestion
```prisma
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 13]
```diff
@@ -7,13 +7,59 @@ summary: Learn how to use TiDB Cloud serverless driver with Prisma ORM.
...
+  provider     = "mysql"
+  url          = env("DATABASE_URL")
+}
+```
+
+## Initialize Prisma Client
+
+Before using Prisma Client, you need to initialize it with `@tidbcloud/prisma-adapter`. For example:
+
+```
```
Comment (Oreoxmt) [inline tidb-cloud/serverless-driver-prisma-example.md:46]:
```suggestion
```js
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 14]
```diff
@@ -216,4 +262,7 @@ To complete this tutorial, you need the following:
    
 ## Use the Prisma adapter in edge environments
 
-Currently, `@tidbcloud/prisma-adapter` is not compatible with edge environments such as Vercel Edge Function and Cloudflare Workers. However, there are plans to support these environments. For more information, see [#21394](https://github.com/prisma/prisma/issues/21394).
\ No newline at end of file
+You can use `@tidbcloud/prisma-adapter` v5.11.0 or a later version in edge environments such as Vercel Edge Function and Cloudflare Workers.
+
+- [Vercel Edge Function example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-vercel-example)
+- [Cloudflare Worker example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-cloudflare-worker-example)
```
Comment (Oreoxmt) [inline tidb-cloud/serverless-driver-prisma-example.md:268]:
```suggestion
- [Cloudflare Workers example](https://github.com/tidbcloud/serverless-driver-example/tree/main/prisma/prisma-cloudflare-worker-example)
```

File: tidb-cloud/serverless-driver-prisma-example.md [hunk 15]
```diff
@@ -216,4 +262,7 @@ To complete this tutorial, you need the following:
    
 ## Use the Prisma adapter in edge environments
 
-Currently, `@tidbcloud/prisma-adapter` is not compatible with edge environments such as Vercel Edge Function and Cloudflare Workers. However, there are plans to support these environments. For more information, see [#21394](https://github.com/prisma/prisma/issues/21394).
\ No newline at end of file
+You can use `@tidbcloud/prisma-adapter` v5.11.0 or a later version in edge environments such as Vercel Edge Function and Cloudflare Workers.
```
Comment (Oreoxmt) [inline tidb-cloud/serverless-driver-prisma-example.md:265]:
```suggestion
You can use `@tidbcloud/prisma-adapter` v5.11.0 or a later version in edge environments such as Vercel Edge Functions and Cloudflare Workers.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request closed without merging**